lib/core/utils/preload-cssom.js

```diff
@@ -60,14 +93,16 @@ function loadCssom({ root, shadowId }, timeout, convertTextToStylesheetFn) {
 					resolve({
 						sheet,
 						isExternal: false,
-						shadowId
+						shadowId,
+						root
 					})
 				);
 				return;
 			}
 
 			// if any import rules exists, fetch via `href` which eventually constructs a sheet with results from resource
 			importRules.forEach(rule => {
+				getExternalStylesheet;
```

Comment:
What's going on here?

Reply:
Good catch. Should be removed, doing so now.